src/modules/aws_ecs_fargate@0.0.1/index.ts

```diff
@@ -145,7 +145,7 @@ export const AwsEcsFargateModule: Module = new Module({
   mappers: {
     cluster: new Mapper<AwsCluster>({
       entity: AwsCluster,
-      entityId: (e: AwsCluster) => e?.clusterArn ?? 'default',
+      entityId: (e: AwsCluster) => e.clusterArn ?? e.id?.toString() ?? '',
```

Comment:
why do we default to `''`?

Reply:
Because I can't convince Typescript that the cloud-derived cluster entity is guaranteed to have the `e.clusterArn` when it won't have an `id` and vice-versa for the DB-derived entity.